src/dp.rs

```diff
@@ -19,50 +62,66 @@ pub trait DifferentialPrivacyBudget {}
 /// Marker trait for differential privacy scalar noise distributions.
 pub trait DifferentialPrivacyDistribution {}
 
-/// Zero-concentrated differential privacy (zCDP) budget as defined in [[BS16]].
+/// Zero-concentrated differential privacy (ZCDP) budget as defined in [[BS16]].
```

Comment:
No, zCDP is yet another definition of DP that works particularly well for Gaussian noise (it uses divergences instead of just probability ratios). It is separate from Apprxomate DP (aka Epsilon-Delta DP) and Pure DP (Epsilon-Delta DP with delta = 0).

Reply:
Got it. Given the huge variety of DP definitions, we should keep in mind that trying to express all of them in an API, regardless of the language, would probably result in more complexity than we can reasonably maintain. I think we're committed to this one, but we need to be very mindful going forward.